src/Evolution/Systems/NewtonianEuler/TurbulenceComputeItems.hpp

```diff
@@ -0,0 +1,199 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/DataBox/DataBoxTag.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Evolution/Systems/NewtonianEuler/Tags.hpp"
+#include "Utilities/TMPL.hpp"
+
+namespace gsl {
+template <typename>
+struct not_null;
+}  // namespace gsl
+
+namespace NewtonianEuler {
+//@{
+/*!
+ * Compute the internal energy density, \f$\rho \epsilon\f$,
+ * where \f$\rho\f$ is the mass density, and \f$\epsilon\f$ is the
+ * specific internal energy.
+ */
+template <typename DataType>
+void internal_energy_density(
+    gsl::not_null<Scalar<DataType>*> result,
+    const Scalar<DataType>& mass_density,
+    const Scalar<DataType>& specific_internal_energy) noexcept;
+
+template <typename DataType>
+Scalar<DataType> internal_energy_density(
+    const Scalar<DataType>& mass_density,
+    const Scalar<DataType>& specific_internal_energy) noexcept;
+//@}
+
+//@{
+/*!
+ * Compute the kinetic energy density, \f$\rho v^2/2\f$,
+ * where \f$\rho\f$ is the mass density, and \f$v\f$ is the
+ * magnitude of the velocity.
+ */
+template <typename DataType, size_t Dim, typename Fr>
+void kinetic_energy_density(
+    gsl::not_null<Scalar<DataType>*> result,
+    const Scalar<DataType>& mass_density,
+    const tnsr::I<DataType, Dim, Fr>& velocity) noexcept;
+
+template <typename DataType, size_t Dim, typename Fr>
+Scalar<DataType> kinetic_energy_density(
+    const Scalar<DataType>& mass_density,
+    const tnsr::I<DataType, Dim, Fr>& velocity) noexcept;
+//@}
+
+//@{
+/*!
+ * Compute the local Mach number, \f$\text{Ma} = v/c_s\f$,
```

Comment:
[optional] I think the `/*!` need a `\brief` on the first line if you want them to have a brief comment, but I wasn't able to render on my machine (for unrelated reasons)